roles/openshift_storage_glusterfs/files/v3.7/heketi-template.yml

```diff
@@ -114,10 +116,10 @@ parameters:
   displayName: heketi executor type
   description: Set the executor type, kubernetes or ssh
   value: kubernetes
-- name: HEKETI_KUBE_NAMESPACE
```

Comment:
Why was the namespace removed?

Reply:
Just to be informational: This was just specifying the namespace the pod was running in. We changed the container such that we can detect the namespace through kubeapi rather than having to have it specified inside the container.